ui-charts/src/spaceTimeChart/utils/canvas.ts

```diff
@@ -348,27 +348,19 @@ export type CanvasRect = {
 
 export function fillRect(
   ctx: CanvasRenderingContext2D,
-  rect: CanvasRect,
-  spaceTimeContext: SpaceTimeChartContextType
+  { timeStart, timeEnd, spaceStart, spaceEnd }: CanvasRect,
+  { getPoint }: SpaceTimeChartContextType
 ) {
-  const { getTimePixel, getSpacePixel, timeAxis } = spaceTimeContext;
-  const { timeStart, timeEnd, spaceStart, spaceEnd } = rect;
+  const topLeft = getPoint({ time: Number(timeStart), position: spaceStart });
+  const bottomRight = getPoint({ time: Number(timeEnd), position: spaceEnd });
```

Comment:
Nit : Maybe it's about preferences but i'm feeling a bit weird with those const names as i expect topSomething to be greater than bottomAnotherStuff but you end up by soustracting top to bottom so in my mind it will in first read return something negative.

maybe we could use 

```
const startPoint = getPoint({ time: Number(timeStart), position: spaceStart });
const endPoint = getPoint({ time: Number(timeEnd), position: spaceEnd });
```

Reply:
The rectangle can be drawn upside down.  I also think `startPoint`/`endPoint`  would make more sense here.

Reply:
Right… TBH, I don't find `startPoint` and `endPoint` to be any more descriptive than `pointA` and `pointB`, but I don't really care that much either way. Changed!